**Due**:Tuesday, June 3, 2021 11:59 PM EST

**Author**:Alejandro M. Ouslan
## Exercice 1:
- Two soccer (Football) teams go into a penalty shoot-out. Each team selects their five
players for the shootout. The probabilities of them making their individual, independent
shot is as follows:
    - Team A: {0.89, 0.67, 0.78, 0.44, 0.75}
    - Team B: {0.91, 0.75, 0.39, 0.60, 0.88}

In [29]:
# imports 
import polars as pl
import numpy as np
from scipy.stats import norm, binom 

In [2]:
team_a = np.array([0.89, 0.67, 0.78, 0.44, 0.75])
team_b = np.array([0.91, 0.75, 0.39, 0.60, 0.88])

## Exercice 1.a:
- What is the probability of each team’s five players scoring their penalty? Which team
has the higher probability?

In [3]:
print(team_a.mean())
print(team_b.mean())

0.706
0.7060000000000001


- **Answer**: Both teams have the same probabily to score.

## Exercice 1.b:
- Suppose the second player in Team A misses, and so does the third player of Team B; what is the probability of the remaining four players in each team scoring their penalty?


In [10]:
miss_a = np.delete(team_a, 1)
miss_b = np.delete(team_b, 2)
print(miss_a.mean())
print(miss_b.mean())

0.715
0.785


- **Answer**: Team A now has a probability of $0.715$ and Team B now has a probability of $0.785$

## Exercice 1.c:
- Now, suppose the second player of each team has a conditional probability based on whether the first penalty taker scores their penalty (assume the probability of the first penalty taker stays the same):
  - For Player 2 in Team A:
  $$f_{1|2}(1|1) = 0.83,f_{1|2}(0|1) = 0.17$$
  $$f_{1|2}(1|0) = 0.60,f_{1|2}(0|0) = 0.40$$

  - For Player 2 in Team B:
  $$f_{1|2}(1|1) = 0.81,f_{1|2}(0|1) = 0.19$$
  $$f_{1|2}(1|0) = 0.72,f_{1|2}(0|0) = 0.28$$

- Assuming the remaining players are independent of players one and two, what is the
probability of the whole team if:

## Exercice 1.d:
  - The first shooter from both teams scores. Which team has the higher probability?

In [16]:
hit_a = np.delete(team_a, 0)
hit_b = np.delete(team_b, 0)

hit_a[hit_a == 0.67] = 0.83
hit_b[hit_b == 0.75] = 0.81

print(hit_a.mean())
print(hit_b.mean())

0.7
0.67


- **Answer**: Team A has the higher probability with 0.7

## Exercise 1.e:

- Team A’s first player scores, but the second player does not.

In [18]:
miss_a = np.delete(team_a, 0)
miss_a = np.delete(team_a, 0)

print(miss_a.mean())

0.66


**Answer**: The probability of the remading players will be 0.66

# Excercise 2
Let X be a random variable distributed as Normal (6,3). Find the probabilities of the following events:

1. $P(X <= 6)$
2. $P(X > 4)$
3. $P(|X - 5| > 1)$

In [28]:
mean, std = 6, 3
print(norm(loc=mean, scale=std).cdf(6))
print(1 - norm(loc=mean, scale=std).cdf(4))


0.5
0.7475074624530771


In [25]:
upper = norm(loc=mean, scale=std).cdf(6)
lower = norm(loc=mean, scale=std).cdf(4)

print(1 - (upper - lower))

0.7524925375469229


**Answers**:

1. $P(X <= 6) = 0.5$
2. $P(X > 4) = 0.7475$
3. $P(|X - 5| > 1) = 0.7525$

# Excercise 3

- Just prior to jury selection for O. J. Simpson’s murder trial in 1995, a poll found that about 20% of the adult population believed Simpson was innocent (after much of the physical evidence in the case had been revealed to the public). Ignore the fact that this 20% is an estimate based on a subsample from the population; for illustration, take it as the true percentage of people who thought Simpson was innocent prior to jury selection Assume that the 12 jurors were selected randomly and independently from the population (although this turned out not to be true).
    - Find the probability that the jury had at least one member who believed in Simpson’s innocence prior to jury selection. [Hint: Define the Binomial (12,20) random variable X to be the number of jurors believing in Simpson’s innocence].

In [34]:
# Calculating the Binomial CDF for n >= 1
k = 1
n = 12
p = .2
1 - binom.cdf(k, n, p)  + binom.pmf(k, n, p)

0.931280523264

**Answer**: The probability that at least 1 jury found Simpsone innocent is $0.9313$

- Find the probability that the jury had at least two members who believed in Simpson’s innocence. [Hint: $𝑃(𝑋 ≥ 2) = 1 − 𝑃(𝑋 ≤ 1)$ and $𝑃(𝑋 ≤ 1) = 𝑃(𝑋 = 0) + 𝑃(𝑋 = 1)$.]

In [35]:
# Calculating the Binomial CDF for n >= 2
k = 2
n = 12
p = .2
1 - binom.cdf(k, n, p)  + binom.pmf(k, n, p)

0.7251220930560002

**Answer**: The probability that at least 1 jury found Simpsone innocent is $0.9313$

# Excercice 4
Let X denote the annual salary of university professors in the United States, measured in
thousands of dollars. Suppose that the average salary is 52.3, with a standard deviation of
14.6. Find the mean and standard deviation when salary is measured in dollars.

In [36]:
# transforming from thousans of dollars to dollars 
print(52.3*1000)
print(14.6*1000)

52300.0
14600.0


**Answer**: The mean salary is $52300$ and the standard deviation is $14600$